### DISCUSSION TITLE
[Feature] "Recently imported" - view

### DISCUSSION THREAD

**Discussion by blublub** [immich-app/immich#10398] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi

I would like to suggest a "new view" option: "recently imported" which shows all assets which have benn "recently" imported ( changeable time via slider - 5m, 30m, 60m, today, 3day, 1w)

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by vmoriaux** [comment]
This could be just another filter in the regular search, assuming the add date is persisted.

To work around the lack of this feature, I create an "import" album and drop my picture into it. If that works well for manual uploads from the webui, that doesn't cover other import methods.

**Reply by blublub** [reply]
Another filter would be also "ok" - however at some point the search filter will get very crowded and be hard to use

Another idea I have would be "smart albums" - so albums which work / get filled by a filter:
by date taken
recently imported- x- days / from camery Xyz
duplicates - as an overview
most viewed

**Comment by bo0tzz** [comment]
#8716
